README.md

```diff
@@ -1 +1,121 @@
+
+
+If a player fails to catch the hammer, they must drink and immediately end their turn, passing the hammer to the next play in turn order.
+
+
+#### Strike
+Upon catching the hammer, the player must immediately perform a strike.
+The player may not hesitate or aim, though they are allowed to bring them hammer up and down again after catching.
+
+
+If that player is a Lad, they must strike their starting nail.
+If they are a Lord, they must strike one of the center nails.
+
+
+If a Lad hammers their starting nail completely into the stump, they instantly ascend to the rank of Lord. 
+This can happen between strikes.
+I.e. if a Lad has two strikes because he or she performed a double hammer flip, and they drive their starting nail completely into the stump on the first hit, the next hit must be performed on a center nail as Lord.
+
+
+A nail is considered completely hammered into the stump if all plays agree or if the Hammer Test decides it is (see Hammer Test). Any player may challenge a nail and perform a Hammer Test. This effectively pauses the game. Giving away the hammer for someone to perform the Hammer Test does not end your turn, and that player must return the hammer to you upon completing the Hammer Test, resuming the game.
+
+
+Each successful strike on a nail grants the player one drink to give out during the Drink phase. 
+Each miss is one drink the player must take themselves during the Drink phase.
+
+
+If a player hits the wrong nail it counts as a miss. If doing so results in the final center nail being hammered into the stump, A new center nail is created and the current player is removed from the game.
+
+
+#### Drink
+A "Drink" is a consistent unit which must be decided upon before the game begins.
+
+
+During the drink phase the player may give out drinks for each successful strike they made during the Strike phase. 
+For each strike made as a Lad, that player may give a drink out to another Lad. For each strike made as a Lord, that player may give a drink out to another Lord or make all remaining Lads drink at once.
+
+
+The final strike of a starting nail is considered a strike made as a Lord, as that player becomes a Lord simultaneously to striking the nail.
+
+
+The final strike of the game (the last center nail) is considered a strike made as a rank even higher than Lord. In this case all other players must drink.
+
+
+The player then must drink for each missed strike he or she performed during the Strike phase.
+
+
+Drinks may only be given out as long as the current player holds the hammer. If the player gives away the hammer without giving out drinks, their turn ends immediately and they forfeit giving out those drinks. However, they still must drink for each missed strike they made. Giving away the hammer for a hammer test does not count as forfeiting the hammer, and the player performing the hammer test must return the hammer when they are finished.
```

Comment:
Maybe capitalize Hammer Test here, since it has been capitalized in other places so far.